packages/terra-table/src/subcomponents/ColumnResizeHandle.jsx

```diff
@@ -35,17 +40,24 @@ const propTypes = {
    */
   height: PropTypes.number.isRequired,
 
+  /**
+   * String that specifies the init height for the resize handler to accommodate actions row.
+   */
+  resizeHandlerInitHeight: PropTypes.string,
```

Comment:
nit: Can we just call this `initialHeight` so we don't have to abbreviate to 'init'? It can be assumed that the height is for the resize handler since it's on the ColumnResizeHandle component.

Reply:
Renamed in https://github.com/cerner/terra-framework/pull/2024/commits/111b1e8f2eb8abc3d6140ffda8cf602404f0fc2d